clients/admin-ui/src/types/api/models/ConnectionConfigurationResponse.ts

```diff
@@ -22,7 +22,7 @@ export type ConnectionConfigurationResponse = {
   last_test_timestamp?: string | null;
   last_test_succeeded?: boolean | null;
   saas_config?: SaaSConfigBase | null;
-  secrets?: null;
+  secrets?: any;
```

Comment:
same here, this was causing some issues so I reverted it back to any as a quick workaround

Reply:
@erosselli can you explain in more detail what issues you were running into? I'm wondering if this is pydantic related